capa-controller-role/trusted-entities.json

```diff
@@ -4,7 +4,7 @@
         {
             "Effect": "Allow",
             "Principal": {
-                "AWS": "arn:*:iam::${AWS_ACCOUNT}:user/${INSTALLATION_NAME}-capa-controller"
+                "AWS": "arn:aws:iam::084190472784:root"
```

Comment:
AFAIK it was wrong, can you verify?

Reply:
this is wrong this will fail for china

Reply:
but if I leave it like before, it is different from what we have in TF or CF template -> https://github.com/giantswarm/giantswarm-aws-account-prerequisites/blob/master/capa-controller-role/variables.tf#L5
if it works and it has lower privileges, then I take it as the default principal customer need to set

Reply:
but it also means the AWS account should be the one where the capa controller run, MC account, right?

Reply:
or do we need the other for our staff?